server/exposures/cmaq.py

```diff
@@ -122,33 +98,68 @@ def get_scores(self, **kwargs):
                       'central': 'Central',
                       'mountain': 'Mountain',
                       'pacific': 'Pacific'}
+        # validate input from user
         is_valid, message = self.validate_parameters(**kwargs)
         if not is_valid:
             return message
-
-        if kwargs.get('resolution') == 'hour':
-            time_delta = 1
+        # determine exposure type to query
+        if kwargs.get('exposureType') == 'pm25':
+            exposure = 'pmij'
         else:
-            time_delta = 24
-
+            exposure = 'o3'
+        # set resolution and aggregate to query
+        if kwargs.get('resolution') == 'day':
+            exposure += '_' + kwargs.get('aggregation') + '_24'
+        elif kwargs.get('resolution') == '7day':
+            exposure += '_' + kwargs.get('aggregation') + '_7day'
+        elif kwargs.get('resolution') == '14day':
+            exposure += '_' + kwargs.get('aggregation') + '_14day'
+        # create data object
         data = {}
         data['scores'] = []
+        # set UTC offset as time zone parameter for query
         if kwargs.get('utcOffset') == 'utc':
             tzone = pytz.timezone(tzone_dict.get(kwargs.get('utcOffset')))
         else:
             tzone = pytz.timezone('US/' + tzone_dict.get(kwargs.get('utcOffset')))
-
+        # datetime objects for query and output adjustment
+        utc_time = pytz.timezone('UTC').localize(datetime.strptime(kwargs.get('startDate'), "%Y-%m-%d"))
+        start_time = tzone.localize(datetime.strptime(kwargs.get('startDate'), "%Y-%m-%d"))
+        end_time = tzone.localize(datetime.strptime(kwargs.get('endDate'), "%Y-%m-%d"))
+        time_diff = utc_time - start_time
```

Comment:
time_diff should be end_time - start_time rather than utc_time - start_time, right?

Reply:
Probably a poor name choice on my part, it is the time difference between UTC and the chosen time zone by the user. {utc, eastern, central, mountain, pacific}
It is used to reset the utc_date_time results from the query back into the original format as submit by the user.
The range for this would be 0 to 8 hours (utc = 0, pacific = 8)